package.json

```
@@ -41,14 +41,17 @@
     "remark-stringify": "^9.0.0",
     "tape": "^5.0.0",
     "tinyify": "^3.0.0",
+    "type-coverage": "^2.18.0",
+    "typescript": "^4.3.4",
```

Where possible, unified uses broader version ranges
```suggestion
    "type-coverage": "^2.0.0",
    "typescript": "^4.0.0",
```